core/domain/email_manager.py

```diff
@@ -1732,6 +1732,21 @@ def send_account_deleted_email(user_id, user_email):
         recipient_email=user_email)
 
 
+def send_account_deletion_failed_email(user_id, user_email):
+    """Sends an email to user whose account was deleted.
```

Comment:
This seems off? It should be sending an email to the admin because it's due to the failure of the cron job. The admin can then fix it and then when the regular cron runs again, the user will be notified accordingly (right)?

Reply:
Oh, forgot to change the docstring.